rancher/rancher-ui/infrastructure/hosts/custom/index.md

```diff
@@ -40,9 +40,9 @@ http://<rancher-server-ip>:8080/v1/projects/1a5/scripts/<registrationToken>
 
 ### Security Groups/Firewalls 
 
-For any hosts that are added, ensure that any security groups or firewalls allow traffic. If these are not enabled, the Rancher functionality will be limited.
+For any hosts that are added, ensure that any security groups or firewalls allow traffic between the Rancher server and the Rancher agents, otherwise Rancher functionality will be limited.
 
-* From and to all other hosts on UDP ports `500` and `4500` (for IPsec networking)
+* From Rancher server to UDP ports `500` and `4500` (for IPsec networking) on Rancher agent hosts
 
```

Comment:
This is actually not true. I confirmed with the engineers that these ports only need to be on the Rancher agent hosts so they can communicate with each other. 

For Rancher/server, the only thing needed is for the hosts to be able to communicate to the rancher/server and the port that you start Rancher server with (Default is 8080). http://docs.rancher.com/rancher/faqs/troubleshooting/#how-to-check-your-host-registration-is-set-correctly